src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java

```diff
@@ -7,6 +7,8 @@
   public static final String SIERRA_API_UPDATED_DATE = "updatedDate";
   public static final String SIERRA_API_OFFSET = "offset";
   public static final String SIERRA_API_LIMIT = "limit";
+  public static final String SIERRA_API_FIELDS_PARAMETER = "fields";
+  public static final String SIERRA_API_FIELDS_VALUE = "id";
```

Comment:
should we rename the value to SIERRA_API_FIELDS_VALUE_ID? Because if we need some other field in the future, then specifying the field value in the key would make sense.